### PR TITLE
Support for batch processing from QUARKUS-1412

### PR DESCRIPTION
**Pull request by fedinskiy**

### CODE REVIEW

File: QUARKUS-640.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Following actions were taken to ensure test coverage and automation for reactive
  - Kafka consumer with JSON / AVRO body messages.
  - Kafka Streams with JSON body messages and windowed.
  - Tested over Kafka Strimzi and RH AMQ Streams.
+ - Kafka producer and consumer with batch processing
```
Comment (rsvoboda) [inline QUARKUS-640.md:13]: RFE also mentions `Better rebalance handling` and `Checkpointing`

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Please creat QUARKUS-1412.md and cover the additions, it's not clear what are the new pieces without looking into diff and there is no clear mapping by file name to https://issues.redhat.com/browse/QUARKUS-1412

Is there any upstream documentation for the new features, please ask for it on the JIRA

File: QUARKUS-640.md (hunk 1 above)
Comment (fedinskiy) [reply QUARKUS-640.md:13]: I am not sure we can test better rebalance and Checkpointing is not yet implemented

File: QUARKUS-1412.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Getting familiar with the feature
+Following actions were taken to ensure familiarity:
+ - Focus on exploratory testing of the features
+ - Ensure good user experience and simplicity of use
+
+
+## Advanced topics for test development
+The following advanced topics are listed for future consideration. There should be dedicated feature requirements and/or enhancements for these specific areas.
+ - Checkpointing(when it is ready)
```
Comment (rsvoboda) [inline QUARKUS-1412.md:21]:
```suggestion
 - Checkpointing (when it is ready)
```

File: QUARKUS-1412.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# QUARKUS-1412 - Integrate more advanced Kafka features
+
+The goal of this test plan is to cover the main points of conflicts between Quarkus and Kafka. 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1412
+
+## Scope of the testing
+ - Kafka producer and consumer with batch processing
```
Comment (rsvoboda) [inline QUARKUS-1412.md:8]: What about load balancing scenario, do we have any?

File: QUARKUS-640.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ Following actions were taken to ensure test coverage and automation for reactive
  - Kafka consumer with JSON / AVRO body messages.
  - Kafka Streams with JSON body messages and windowed.
  - Tested over Kafka Strimzi and RH AMQ Streams.
```
Comment (rsvoboda) [inline QUARKUS-640.md:12]: Revert QUARKUS-640.md changes

**Review by rsvoboda — CHANGES_REQUESTED** [review]

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**